master/internal/api/apiutils/utils.go

```diff
@@ -51,6 +54,21 @@ var (
 	}
 )
 
+// AddRBACSuffix adds a "check your permission" string to errors if RBAC is enabled.
+func AddRBACSuffix(err error) error {
+	if config.GetAuthZConfig().IsRBACUIEnabled() {
+		e := err.Error()
+		toks := strings.Split(e, "desc =")
```

Comment:
I don't like that this is hard-coded in, but I don't know how else to get just the description parsed out of err.Error().

Reply:
if the utility only operated on the error message and didn't take in the whole error would that be easier? just adds a prefix. or maybe even no argument. `AddRBACSuffix() if rbac return msg else ""`